packages/cli/src/__tests__/commands/Init.test.ts

```diff
@@ -25,137 +22,119 @@ test('works with url param', async () => {
   expect(stripAnsi(result.stdout)).toMatchSnapshot()
 
-  const schema = fs.readFileSync(
-    join(ctx.tmpDir, 'prisma', 'schema.prisma'),
-    'utf-8',
-  )
+  const schema = fs.readFileSync(join(ctx.tmpDir, 'prisma', 'schema.prisma'), 'utf-8')
   expect(schema).toMatch(defaultSchema('sqlite'))
 
   const env = fs.readFileSync(join(ctx.tmpDir, '.env'), 'utf-8')
   expect(env).toMatchInlineSnapshot(`
-# Environment variables declared in this file are automatically made available to Prisma.
-# See the documentation for more detail: https://pris.ly/d/prisma-schema#using-environment-variables
+    # Environment variables declared in this file are automatically made available to Prisma.
+    # See the documentation for more detail: https://pris.ly/d/prisma-schema#using-environment-variables
 
-# Prisma supports the native connection string format for PostgreSQL, MySQL, SQLite, SQL Server and MongoDB (Preview).
-# See the documentation for all the connection string options: https://pris.ly/d/connection-strings
+    # Prisma supports the native connection string format for PostgreSQL, MySQL, SQLite, SQL Server and MongoDB (Preview).
+    # See the documentation for all the connection string options: https://pris.ly/d/connection-strings
 
-DATABASE_URL="file:./dev.db"
-`)
+    DATABASE_URL="file:./dev.db"
+  `)
 })
 
 test('works with provider param - SqlServer', async () => {
   ctx.fixture('init')
   const result = await ctx.cli('init', '--datasource-provider', 'SqlServer')
   expect(stripAnsi(result.stdout)).toMatchSnapshot()
 
-  const schema = fs.readFileSync(
-    join(ctx.tmpDir, 'prisma', 'schema.prisma'),
-    'utf-8',
-  )
+  const schema = fs.readFileSync(join(ctx.tmpDir, 'prisma', 'schema.prisma'), 'utf-8')
   expect(schema).toMatch(defaultSchema('sqlserver'))
 
   const env = fs.readFileSync(join(ctx.tmpDir, '.env'), 'utf-8')
   expect(env).toMatchInlineSnapshot(`
-# Environment variables declared in this file are automatically made available to Prisma.
-# See the documentation for more detail: https://pris.ly/d/prisma-schema#using-environment-variables
+    # Environment variables declared in this file are automatically made available to Prisma.
+    # See the documentation for more detail: https://pris.ly/d/prisma-schema#using-environment-variables
 
-# Prisma supports the native connection string format for PostgreSQL, MySQL, SQLite, SQL Server and MongoDB (Preview).
-# See the documentation for all the connection string options: https://pris.ly/d/connection-strings
+    # Prisma supports the native connection string format for PostgreSQL, MySQL, SQLite, SQL Server and MongoDB (Preview).
+    # See the documentation for all the connection string options: https://pris.ly/d/connection-strings
 
-DATABASE_URL="sqlserver://localhost:1433;database=mydb;user=SA;password=randompassword;"
-`)
+    DATABASE_URL="sqlserver://localhost:5432;database=mydb;user=SA;password=randompassword;"
```

Comment:
https://github.com/prisma/prisma/pull/10984/files#r780439722

Reply:
fixed in [`0a42e7a` (#10984)](https://github.com/prisma/prisma/pull/10984/commits/0a42e7acee6ec3f807f412b4ff62319cdd9ecbca)